src/components/Button.js

```diff
@@ -1,31 +1,89 @@
-import styled, { css } from 'styled-components';
-import '../index.css';
+import styled, { css } from "styled-components";
+import "../index.css";
+import React from "react";
 
-const Button = styled.button`
+const Button = props => {
```

Comment:
You must add some comments for what is passed into the props, and what it returns etc. =) Well-documented code is good code!